pkg/objectstore/s3/s3.go

```diff
@@ -82,5 +84,21 @@ func GetObjLockInfo(backupLocation *stork_api.BackupLocation) (*s3.GetObjectLock
 		}
 		return nil, err
 	}
-	return out, err
+	objLockInfo := &common.ObjLockInfo{}
```

Comment:
While we have brought  the object store changes from px-backup to stork, can we assess how good bad ugly it would be to bring back S3 minio related error handling(_Method not supported_) that occurs for a lower version of minio server( no support for object lock) . It will be something to be added after line 83.

Reply:
Added it. Will take care of remove it in px-backup code.